components/director/internal/domain/runtime/converter.go

```diff
@@ -1,4 +1,55 @@
 package runtime
 
+import "github.com/kyma-incubator/compass/components/director/internal/graphql"
+
 type Converter struct {
 }
+
+func (c *Converter) ToGraphQL(in *RuntimeModel) *graphql.Runtime {
+	return &graphql.Runtime{
+		ID:     in.ID,
+		Status: c.statusToGraphQL(in.Status),
+		//Annotations:in.Annotations, //TODO: Wait for scalar unmarshalling
+		//Labels:in.Labels, //TODO: Wait for scalar unmarshalling
+	}
+}
+
+func (c *Converter) MultipleToGraphQL(in []*RuntimeModel) []*graphql.Runtime {
```

Comment:
![](https://media.giphy.com/media/2vlC9FMLSmqGs/giphy.gif)

Where can I find tests?

Reply:
The tests have been pushed while you were doing the review 😛